hs_rest_api/tests/http/test_file_metadata.py

```diff
@@ -131,6 +131,9 @@ def test_metadata_update_retrieve(self):
             "end": "2018-02-06"
         })
 
+        # check for logical_file in metadata
+        self.assertEqual(json_response.get("logical_file"), {})
+
```

Comment:
I think empty dict evaluates to False so that's an alternate  assert option here

Reply:
That is an alternate way to write the assertion.